platforms/ide/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/ToolingApiRemoteIntegrationTest.groovy

```diff
@@ -92,7 +108,139 @@ class ToolingApiRemoteIntegrationTest extends AbstractIntegrationSpec {
         }
 
         then:
-        toolingApi.gradleUserHomeDir.file("wrapper/dists/custom-dist").assertIsDir().listFiles().size() == 1
+        assertWrapperDirectoryContentDownloaded()
+    }
+
+    def "loads host credentials from gradle.properties"() {
```

Comment:
This is not the right place for the integration tests of this feature. The Tooling API is not directly related to these changes.

I think the right place would be `platforms/core-runtime/wrapper-main/src/integTest/groovy/org/gradle/integtests`.

Reply:
I'm not sure I exactly follow what you mean, but as I see it, we would ideally add extra tests to `WrapperHttpIntegrationTest` and `WrapperHttpsIntegrationTest`. Where now basic authentication is exercised, we would also add tests for bearer authorization.

Or, since http/https and the authorization mechanisms are kind of orthogonal, if you see a better way of organizing things, I'm open to it.

Reply:
@jbartok I'm not sure how to solve this best, I'm just guessing.

Should I move tests from `platforms/ide/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/ToolingApiRemoteIntegrationTest.groovy` to 
`platforms/core-runtime/wrapper-main/src/integTest/groovy/org/gradle/integtests/WrapperHttpsIntegrationTest.groovy`?

a) keep everything where it is

b) move only
* "loads api token from gradle.properties"
* "loads api token from gradle.properties and rejects invalid token"

c) also move
* "loads credentials from gradle.properties"
* "loads host credentials from gradle.properties"
* "loads credentials from gradle.properties and reject invalid password"

Maybe we should create a new test called `WrapperAuthIntegrationTest`?

Reply:
Hi @jbartok,

I tried to implement a new test, please tell me what you think about it.

Reply:
Please also remove all changes from `ToolingApiRemoteIntegrationTest`.